modules/ztp-image-based-upgrade-prep-label-extramanifests.adoc

```diff
@@ -0,0 +1,96 @@
+
+.Prerequisites
+
+* Provision one or more managed clusters with GitOps ZTP.
+* Log in as a user with `cluster-admin` privileges.
+* Generate a seed image from a compatible seed cluster.
+* Create a separate partition on the target cluster for the container images that is shared between stateroots. For more information about, see _Additional resources_.
+* Deploy a version of {lcao} that is compatible with the version used with the seed image.
+
+.Procedure
+
+. If you have manifests that you want to be extracted and applied during the upgrade, label them with the `lca.openshift.io/target-ocp-version: <target_version>` label in your existing `PolicyGenTemplate` CR:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1
+kind: PolicyGenTemplate
+metadata:
+  name: example-sno
+spec:
+  bindingRules:
+    sites: "example-sno"
+    du-profile: "4.15"
+  mcp: "master"
+  sourceFiles:
+    - fileName: SriovNetwork.yaml
+      policyName: "config-policy"
+      metadata:
+        name: "sriov-nw-du-fh"
+        labels:
+          lca.openshift.io/target-ocp-version: "4.15" <1>
+      spec:
+        resourceName: du_fh
+        vlan: 140
+    - fileName: SriovNetworkNodePolicy.yaml
+      policyName: "config-policy"
+      metadata:
+        name: "sriov-nnp-du-fh"
+        labels:
+          lca.openshift.io/target-ocp-version: "4.15" <1>
+      spec:
+        deviceType: netdevice
+        isRdma: false
+        nicSelector:
+          pfNames: ["ens5f0"]
+        numVfs: 8
+        priority: 10
+        resourceName: du_fh
+    - fileName: SriovNetwork.yaml
+      policyName: "config-policy"
+      metadata:
+        name: "sriov-nw-du-mh"
+        labels:
+          lca.openshift.io/target-ocp-version: "4.15" <1>
+      spec:
+        resourceName: du_mh
+        vlan: 150
+    - fileName: SriovNetworkNodePolicy.yaml
+      policyName: "config-policy"
+      metadata:
+        name: "sriov-nnp-du-mh"
+        labels:
+          lca.openshift.io/target-ocp-version: "4.15" <1>
+      spec:
+        deviceType: vfio-pci
+        isRdma: false
+        nicSelector:
+          pfNames: ["ens7f0"]
+        numVfs: 8
+        priority: 10
+        resourceName: du_mh
+    - fileName: DefaultCatsrc.yaml <2>
+      policyName: "config-policy"
+      metadata:
+        name: default-cat-source
+        namespace: openshift-marketplace
+        labels:
+            lca.openshift.io/target-ocp-version: "4.15" <1>
+      spec:
+          displayName: default-cat-source
+          image: quay.io/example-org/example-catalog:v1
+----
+<1> Ensure that the `lca.openshift.io/target-ocp-version` label matches either the y-stream or the z-stream of the target {product-title} version that is specified in the `spec.seedImageRef.version` field of the `ImageBasedUpgrade` CR. The {lcao} only applies the CRs that match the specified version.
+<2> (Optional) If you do not want to use custom catalog sources, remove this entry.
```

Comment:
You can omit "(Optional)" here. "If" indicates that this change is conditional on what the readers wants to do.